medicines/api/README.md

```diff
@@ -59,3 +59,12 @@ To see the graphql explorer, go to http://127.0.0.1:8080/graphiql.
    ```sh
    api-558646c969-9mdxp api [2020-01-20T15:02:57Z INFO  actix_web::middleware::logger] 10.244.1.1:51524 "GET /healthz HTTP/1.1" 200 2 "-" "kube-probe/1.14" 0.000059
    ```
+
+## Deploy API pod in Kuberbetes cluster via CI/CD pipeline
+
+We are using a [secondary pipeline](./azure-pipeline.yml) instead of the [main pipeline](../../azure-pipelines.yml) because we have created a new development environment that enables us to create and test kubernetes features.
```

Comment:
link is given as `.yml` but new file is `.yaml` (maybe rename new file to `.yml` to match top-level one)

Reply:
Good catch thanks

Reply:
Looks like the rename hasn't been pushed

Reply:
The rename of the file caused the deletion of the pipeline on azure 😭

Reply:
I rollback to the previous name and now it’s working again

Reply:
Please, remember this pipeline is temporary

Reply:
I tried to resolve the Azure issue but didn't have permissions to edit the existing one